mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -106,6 +106,17 @@ private[ml] trait HasProbabilityCol extends Params {
   def getProbabilityCol: String = get(probabilityCol)
 }
 
+private[ml] trait HasFitIntercept extends Params {
+  /**
+   * param for fitting the intercept term
+   * @group param
+   */
+  val fitIntercept: BooleanParam = new BooleanParam(this, "fitIntercept", "fits the intercept term or not")
```

Comment:
"fits the intercept term or not" --> "Indicates whether to fit an intercept term"